fm.reaper.Reaper.yml

```diff
@@ -0,0 +1,62 @@
+app-id: fm.reaper.Reaper
+runtime: org.freedesktop.Platform
+runtime-version: '21.08'
+sdk: org.freedesktop.Sdk
+tags:
+ - proprietary
+command: /app/REAPER/reaper
+
+finish-args:
+  - --share=ipc
+  - --share=network
+  - --socket=x11
+  - --socket=pulseaudio
+  - --filesystem=xdg-run/pipewire-0
```

Comment:
Does Reaper use JACK?

Reply:
Yes it can.

Reply:
Yes, it can use.
![image](https://user-images.githubusercontent.com/54215258/170916565-d89db4de-703b-4448-9ccf-f9530b760e1f.png)